REB_firmware/REB_firmware_update/v0/producer_REB_firmware_update.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+from ccsTools import ccsProducer
+from java.lang import Exception
+import Tkinter
+import glob
+import os
+import subprocess
+
+CCS.setThrowExceptions(True);
+
+#Copy firmware_load_scripts files
+instdir = os.getenv('LCATR_INSTALL_AREA').replace('share/','')
+os.system('cp -vp %s/REB_v5/firmware_load_scripts/bin/et_prog_flash.sh .', instdir)
+os.system('cp -vp %s/REB_v5/firmware_load_scripts/bin/flash_load.tcl .', instdir)
+
+#Safety button
+apptxt = "Click here when ready to start firmware update of the REB"
+print apptxt
+top = Tkinter.Tk()
+A = Tkinter.Button(top, text = apptxt, command = top.destroy, bg = "yellow", font = ("Helvetica",24))
+A.pack()
+top.title('Are you ready to start the firmware update?')
+top.mainloop()
+
+#Get list of files in firmware directory
+list_of_firmware_files = glob.glob(os.path.join(instdir,'/REB_v5/targets/REB_v5_top/images/', '*.mcs'))
+
+#Find most recent file
+recent_file = max(list_of_firmware_files, key=os.path.getctime)
```

Comment:
We (by we I mean BNL administration, not really me) don't want the operators to have a way to select firmware through the eTraveler. There should not be any mechanism present for them to make firmware selections (similar to how we only let specific technicians power the REBs on/off, so there is no mechanism in the eTraveler for the operator has no way to power the REBs). 

Is there a way to achieve what you are looking for that would not let the operator have a choice of firmware?